composer.json

```diff
@@ -12,7 +12,9 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0"
+        "php": ">=5.3.0",
+        "psr/container": "^1.0",
+        "phpunit/phpunit": "5.0.0"
```

Comment:
Pinning the 5.0.0 release with an exact match is a total no-go. this version is totally outdated and many bug have been fixed since then.

Thus, the constraint must also allow using PHPUnit 4.8, as testing must go back to PHP 5.3 currently.
Thus, using PHPUnit through composer does *not* belong to this PR (and was rejected in the past)